### ISSUE TITLE
HELP? This site can’t be reached - umap.openstreetmap.fr unexpectedly closed the connection.

### ISSUE THREAD

**Issue by LMCCUBB** [umap-project/umap#1464]
Can't access the umap.openstreetmap.fr website at all.
Error message: This site can’t be reached

Can't view our map, can't log in - nothing.

http://umap.openstreetmap.fr/en/map/volunteering-in-rct_959135

I'm not a developer or trained in any way, please explain in layman's terms.  This is a map we created and is embedded on our site - can't find any information .

#Is the website down?  Is there a security breach?

When I try to navigate to log in page, I get various messages such as:

{"html": "\n\n\n  <h5>Please choose a provider</h5>\n  <div>\n    <ul class=\"login-grid block-grid\">\n      \n        <li>\n          <a rel=\"nofollow\"\n             href=\"/login/github/\"\n             class=\"umap-login-popup login-github\"\n             title=\"Github\"></a>\n        </li>\n      \n        <li>\n          <a rel=\"nofollow\"\n             href=\"/login/bitbucket/\"\n             class=\"umap-login-popup login-bitbucket\"\n             title=\"Bitbucket\"></a>\n        </li>\n      \n        <li>\n          <a rel=\"nofollow\"\n             href=\"/login/twitter-oauth2/\"\n             class=\"umap-login-popup login-twitter-oauth2\"\n             title=\"Twitter-Oauth2\"></a>\n        </li>\n      \n        <li>\n          <a rel=\"nofollow\"\n             href=\"/login/openstreetmap/\"\n             class=\"umap-login-popup login-openstreetmap\"\n             title=\"Openstreetmap\"></a>\n        </li>\n      \n    </ul>\n  </div>\n\n"}

**Comment by Xonnext** [comment]
Same (?) problem here:

Red popup:
Issue reaching that URL (network problem or CORS protection): ...

It is on all the maps..

**Comment by LMCCUBB** [comment]
> Same (?) problem here:
> 
> Red popup: Issue reaching that URL (network problem or CORS protection): ...
> 
> It is on all the maps..

Ahh thank you for replying - hopefully it's a temporary maintenance issue then?  Wish there was an update about it somewhere

**Comment by Xonnext** [comment]
Problem seems to be solved... THANKS

**Comment by yohanboniface** [comment]
Yep, sorry, that was a outage unrelated to uMap itself, but in the same server as the http proxy.
